Packs/Azure-Enrichment-Remediation/ReleaseNotes/1_1_13.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Azure - Network Security Group Remediation
+
+- Added the *SubscriptionID* and *ResourceGroup* optional inputes in case the subscription ID and/or resource group needs to be specified.
```

Comment:
```suggestion
- Added the *SubscriptionID* and *ResourceGroup* optional inputs in case the subscription ID and/or resource group needs to be specified.
```